.github/workflows/nodejs.yml

```diff
@@ -11,4 +11,5 @@ jobs:
     # Documentation: https://github.com/zakodium/workflows#nodejs-ci
     uses: zakodium/workflows/.github/workflows/nodejs.yml@nodejs-v1
     with:
-      node-version-matrix: '[14, 16, 18]'
+      node-version-matrix: '[18,20,22]'
```

Comment:
```suggestion
```

We now usually just inherit the default matrix, which contains all supported Node.js versions.